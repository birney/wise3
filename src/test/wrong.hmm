HMMER2.0
NAME  SEED
LENG  79
ALPH  Amino
RF    no
CS    no
MAP   yes
COM   hmmbuild -f HMM SEED
COM   hmmcalibrate --seed 0 /nfs/disk100/pubseq/Pfam/NEWDB/SH2/HMM
NSEQ  58
DATE  Thu Feb  4 21:23:04 1999
CKSUM 2627
XT      -8455     -4  -1000  -1000  -8455     -4  -8455     -4 
NULT      -4  -8455
NULE     595  -1558     85    338   -294    453  -1158    197    249    902  -1085   -142    -21   -313     45    531    201    384  -1998   -644 
EVD    -7.995505   0.798968
HMM        A      C      D      E      F      G      H      I      K      L      M      N      P      Q      R      S      T      V      W      Y    
         m->m   m->i   m->d   i->m   i->i   d->m   d->d   b->m   m->e
          -33      *  -5459
     1  -7236  -6151  -7608  -1460    211  -7493  -3675  -6106  -7520  -1974  -5505  -6101  -7346  -6241  -6882  -6742  -7084  -6270   6026   1349     1
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10599 -11641   -894  -1115   -701  -1378  -1033  -7285 
     2  -1694  -5990  -7523  -7809   3279  -7334  -3682    282  -7382  -5323  -5353   -320  -7232  -6178  -6788  -6576  -6786  -5989  -2936   3857     2
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10599 -11641   -894  -1115   -701  -1378  -7319  -7276 
     3  -1302     98   -302  -4672   2259  -4519   4007   -858  -4269  -1175   1007  -1278  -4569  -3894  -4073  -1737  -2913   -213   1776   1102     3
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10599 -11641   -894  -1115   -701  -1378  -7319  -7267 
     4  -1696  -5051  -1085  -1251  -5397   3225  -3063  -5159    694  -5078  -4201  -2854     22  -2636    677  -1556  -3503  -4700  -5232  -4522     4
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10599 -11641   -894  -1115   -701  -1378  -7319  -7257 
     5     16  -4184    262   -814    686  -2174    695  -4255   2139  -1932  -3273   1337    235   -780   -442    312   -973  -1583  -4367    294     5
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10599 -11641   -894  -1115   -701  -1378  -7319  -7248 
     6  -3550  -3310  -5950  -5356  -3371  -5229  -4181   3204  -4991   1046   1310   -771  -5242  -4638  -4821  -2099  -1439    979  -4012  -3662     6
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10599 -11641   -894  -1115   -701  -1378  -7319  -7238 
     7  -2712  -4184   -952   -320  -4505    -53  -2345  -4255   -151  -1260     -7    122    424    329  -2433   2301   1640  -3806  -4368  -3685     7
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10599 -11641   -894  -1115   -701  -1378  -7319  -7229 
     8  -2074  -5958  -6742  -4882  -7054    976  -3718  -6264    677  -5853  -5216  -4634  -5822  -3300   3805  -5334  -5090  -6045   -101  -5632     8
     -   -150   -501    233     42   -373    399    104   -628    211   -467   -722    276    392     44    100    359    116   -365   -296   -251 
     -   -212  -3659  -4304  -1684   -538   -701  -1378  -7319  -7219 
     9    647  -4118   -298   1003  -4439   -788    741  -1079   1215  -4134  -3207    215  -1589   1630   -317    470     -4   -623    802  -3619    14
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10524 -11566   -894  -1115   -243  -2690  -7319  -7209 
    10  -1509  -4188   1622   2632  -4509  -1418  -2347   -667    267  -4204  -3278  -1373  -3781   2005  -1030  -1440  -2654  -3810  -4372   -559    15
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10599 -11641   -894  -1115   -701  -1378  -7319  -7200 
    11   3325  -4216  -6178  -5845  -6062  -4560  -5158  -5689  -1206  -5929  -5126  -4938  -5244  -5024  -1017   -446  -2008    749  -6204  -6003    16
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10599 -11641   -894  -1115   -701  -1378  -7319  -7190 
    12  -1811  -4188  -1167   3438   -935  -3728     73  -4223  -1996  -4194  -3284   -657  -3829    375  -2502  -2650  -1029   -811  -4383   -529    17
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10599 -11642   -894  -1115   -701  -1378  -7319  -7180 
    13   -496  -4184    676   1029  -4505  -1418    679  -4256   1517  -4200   -300    198  -3778   1201   1830   -183   -513  -1882  -4367   -868    18
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10599 -11642   -894  -1115   -701  -1378  -7319  -7170 
    14   -633  -2957  -4427  -3832  -2932  -4370    432    156   -476   2332   1704  -3697  -4430   1065    326  -1384  -2885  -2383  -3393    519    19
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -11 -10600 -11642   -894  -1115   -701  -1378  -7319  -7160 
    15  -6143  -5498  -8565  -8001  -3625  -8435  -7123   -268  -7843   3292  -2382  -8182  -7357  -6530  -7298  -7874  -5985  -1460  -5506  -5762    20
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -12 -10600 -11642   -894  -1115   -701  -1378  -7319  -7150 
    16  -2729  -4016   -928  -2106  -4264  -1062  -2402  -1320    896   1300   2084    281  -3826   1390   1151   -481    -78  -1536  -4244    111    21
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -49 -10600  -5282   -894  -1115   -701  -1378  -7319  -7140 
    17    267  -4150   -221    505  -4471   -122  -2311  -1514    507   -860   -622   1535   -501   1422   -287    677    -38  -1869  -4334   -287    22
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -  -1583 -10562   -603   -894  -1115  -1507   -626  -7319  -7129 
    18   -109  -2533  -1496    134  -2721   1006  -1191    102   -852  -2495  -1669  -1223   2296   -798     26    619  -1402  -2084  -2815   1321    23
     -   -149   -483    232     46   -381    398    105   -624    210   -467   -714    275    393     45     95    358    120   -370   -295   -250 
     -   -169  -3284 -10036  -1922   -442  -1039   -962  -7319  -7119 
    19   -168  -3624   -295   1199    -68   1871  -1823  -1063   -354  -3635   -461  -1803  -3256    291    846   -810    564   -165  -3816  -3141    28
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -13  -9980 -11023   -894  -1115   -504  -1762  -7319  -7109 
    20  -1395  -4051   -278   -723  -4371   1575  -2211  -4122   1514   -681    767   1742    550    344   -150  -1256   -852  -1749  -4234  -3551    29
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -12 -10448 -11490   -894  -1115  -1525   -616  -7319  -7098 
    21  -1303  -4079  -2464    700  -1602  -3589   1433   -694   1661  -4094    597   -972   1666    751   1030   -190    142  -1149  -4265  -3584    30
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -12 -10489 -11531   -894  -1115   -186  -3045  -7319  -7087 
    22  -1296  -4184   2519    951  -4505  -3685    848  -4256   -726  -1766  -3273   1051    594   -300    418   -432    122   -826  -4367  -3685    31
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -12 -10600 -11642   -894  -1115   -701  -1378  -7319  -7077 
    23  -5971    427  -7965  -8333  -7548   3798  -7312  -8622  -8266  -8444  -7920  -7298  -6954  -7936  -7776  -6253  -6413  -7608    623  -7381    32
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -12 -10600 -11642   -894  -1115   -701  -1378  -7319  -7066 
    24    657    -23   1098   -699  -4418   -941   -364  -4149   -290   -417   -126  -2348  -3794  -1913  -2458   1590   2151  -3731  -4324  -3658    33
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -12 -10600 -11642   -894  -1115   -701  -1378  -7319  -7055 
    25  -7274  -6182  -7628  -7991   4417  -7490  -3715  -6170  -7556  -5469  -5569  -6138  -7360  -6280  -6916  -6780  -7127  -6327  -2962   1866    34
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -12 -10600 -11643   -894  -1115   -701  -1378  -7319  -7044 
    26  -6133  -5488  -8556  -7989  -3625  -8436  -7117    159  -7831   3112   1066  -8170  -7353  -6525  -7290  -7870  -5974    105  -5506  -5766    35
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -12 -10600 -11643   -894  -1115   -701  -1378  -7319  -7033 
    27   -946  -4360  -7581  -7272  -5013  -7453  -7549   2420  -7258    687  -3691  -7109  -7162  -7212  -7441  -6846  -4873   2920  -6837  -6329    36
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -12 -10601 -11643   -894  -1115   -701  -1378  -7319  -7022 
    28  -8779  -7380  -8290  -8499  -8796  -7334  -7688  -9818  -7760  -9142  -9056  -8481  -7754  -8187   4273  -9233  -8869  -9515  -7500  -8689    37
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -13 -10601 -11643   -894  -1115   -701  -1378  -7319  -7011 
    29  -1430  -4182   1605   2386   1016  -3701  -2364  -1877    554  -1404  -3274  -2340    988   1341  -2456  -2613  -2669  -3799  -4371  -3693    38
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -13 -10601 -11643   -894  -1115   -701  -1378  -7319  -7000 
    30   -811   1038  -6983  -7288  -6924  -4490  -6231  -6742  -6774  -7000  -6027  -5325  -5305  -6315    906   3454  -1341  -5420  -7134  -7122    39
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -13 -10601 -11643   -894  -1115   -701  -1378  -7319  -6989 
    31  -1497  -4184   1338   2327  -4504  -3685   -435  -1510   -466  -1266  -3273   -861  -3778    218    805   1115   -421  -1387  -4367  -3684    40
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -   -229 -10601  -2852   -894  -1115   -701  -1378  -7319  -6977 
    32  -2544  -4015  -2384   -207  -4338   -151    -43  -4087   -120  -4031  -3105    966   -911  -1712   1160   2073   1899  -3638  -4197  -3516    41
     -   -149   -500    233     43   -381    398    105   -627    212   -466   -721    277    393     45     96    360    117   -370   -295   -250 
     -   -233  -3704  -3951    -63  -4547   -114  -3722  -7319  -6966 
    33   -572    309   -784    245   -317  -1172   1400  -1251   1429  -2572  -3199    630  -3705    832   -810    678   1524  -3732  -4294    193    43
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -13 -10518 -11560   -894  -1115   -635  -1490  -7319  -6954 
    34   -550  -4118   -110    249  -4419  -3673  -2333      9    941   -272  -3212  -2316   2854   1226  -1206  -2581   -941   -990  -4313  -3642    44
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -76 -10574  -4563   -894  -1115   -411  -2013  -7319  -6943 
    35  -1439  -5086   -697  -3749  -7091   3546  -4834  -7007  -5343  -7054  -6292   1125  -5310  -4596  -6061  -1596  -4536  -5954  -7225  -6509    45
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -13 -10540 -11582   -894  -1115   -275  -2526  -7319  -6931 
    36   -277    615   2110   1083  -4505    976    321  -4256   -946  -1376  -3273  -1051  -3778    930  -2432   1066   -474  -3806  -4368  -3685    46
     -   -149   -500    233     43   -381    398    105   -627    210   -466   -721    275    396     45     98    360    117   -370   -295   -250 
     -    -45  -5474 -11643  -1505   -626   -701  -1378  -7319  -6919 
    37  -6384  -5646  -7740  -7748   2831  -7543  -4219    635  -7397   -853  -3608  -6441  -7203  -6242  -6861  -6801  -6258  -1299  -3431   4057    50
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -13 -10601 -11644   -894  -1115   -701  -1378  -7319  -6907 
    38   1171   -293  -6607  -6454  -4954  -4640  -5327  -1155  -6079  -4711  -4106  -5174  -5272  -5638  -5815   1859   2339   1756  -5487  -5205    51
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -13 -10602 -11644   -894  -1115   -701  -1378  -7319  -6895 
    39  -5548    726  -8073  -7590    265  -7859  -6933   1805  -7445   2761  -2626  -7548  -7172  -6550  -7142  -7215  -5460    267  -5654  -5732    52
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -14 -10602 -11644   -894  -1115   -701  -1378  -7319  -6883 
    40   -607   1053  -6978  -7296  -6850  -1653  -6236  -6659  -6912  -6927  -5967  -5324  -5300  -6345  -6517   3325   1269  -5383  -7086   -517    53
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -14 -10602 -11644   -894  -1115   -701  -1378  -7319  -6870 
    41  -3195  -3008  -5527  -4904   1429  -1639    -23   -442  -4502    808  -2142  -4377  -4782  -4107  -4296  -3834  -3134   2949   2848  -2915    54
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -14 -10602 -11644   -894  -1115   -701  -1378  -7319  -6858 
    42  -1468    248  -5289  -4655    407  -4516   -283   -988   1333    256    638  -4153  -4566  -3884   2425    -64  -2912    892   1339     25    55
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -14 -10602 -11644   -894  -1115   -701  -1378  -7319  -6845 
    43     -6    776   1857   -226   1013  -3780   1648  -1280   -813  -1628   -340   -865  -3870   -505    211   -788    658   -590   1629    781    56
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -14 -10602 -11644   -894  -1115   -701  -1378  -7319  -6833 
    44  -2711  -4184   1808    311    563   1017  -2343  -1202    727  -4200  -3273   1521  -1611   1203   -208   -676  -2650  -3806   1464  -1375    57
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -14 -10602 -11644   -894  -1115   -701  -1378  -7319  -6820 
    45   -444  -4183   1973   -338  -1263   2075  -2344  -4253    -89  -4198   -177    192  -3778   -545   -302    260  -1633   -176  -4366  -3684    58
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -  -1303 -10602   -773   -894  -1115   -701  -1378  -7319  -6807 
    46  -1654  -3126    892    585  -3447   -681   1000   -197    297  -3142  -2215    839   1662   1550     22  -1534    288   -630  -3309  -2627    59
     -   -153   -497    237     44   -390    408    102   -627    214   -471   -725    275    395     55     92    362    115   -376   -303   -254 
     -  -2642   -353  -4387  -2167   -363    -31  -5570  -7319  -6794 
    47   -971    -13    -81    370  -4479  -1026   1314  -1276   1291  -1680  -3248    646  -3753   1074   1578    140   -359    295  -4342  -3659    68
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -14 -10574 -11616   -894  -1115  -1351   -718  -7319  -6781 
    48  -1452   -432  -5317  -4685    672  -4530  -3411   1174  -4285  -2655  -2016    885   -180  -3912  -1700  -3617  -1465   3088  -3280  -2937    69
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -15 -10574 -11616   -894  -1115   -406  -2028  -7319  -6768 
    49  -2712  -4179  -2561    170  -1276  -1378    783  -1651   2633  -1190  -3269   -721  -3779   1595    924  -1219    376  -3800  -4364     94    70
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -15 -10603 -11645   -894  -1115   -701  -1378  -7319  -6755 
    50  -4519  -5497  -4305  -1190  -5973  -5163   5152  -5554  -2301  -1967  -4630   -909  -1026  -1110   -725  -4396  -4332  -5267    762  -4996    71
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -79 -10603  -4520   -894  -1115   -701  -1378  -7319  -6741 
    51  -4742   2555  -6657  -6333   2540  -6098  -3678   1238  -5936  -4026  -3628   -967  -6082  -5296  -5631  -5238  -4678   -460  -3072   3759    72
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -15 -10539 -11581   -894  -1115   -267  -2566  -7319  -6728 
    52  -2757  -3861   -754  -2226  -4050  -3797   1181    712   1918   -607   -367  -2504  -1264    691   2362  -2720   -284     20  -4128  -3537    73
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -15 -10603 -11645   -894  -1115   -701  -1378  -7319  -6714 
    53  -4514  -4096  -7081  -6634   -723  -6661  -6001   3505    152  -1714  -3321  -6310  -1202  -6188  -6387  -5894  -4487   1567  -5642  -5244    74
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -15 -10603 -11645   -894  -1115   -701  -1378  -7319  -6700 
    54  -2716   1077  -1059   -351   -948  -3699    838   -636   1214   -274    745   1226  -1112   2092   1332  -1343  -2655  -1397  -4330   -270    75
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -15 -10603 -11646   -894  -1115   -701  -1378  -7319  -6687 
    55   -907     54  -2567    146      6  -3690   -192  -4224    970  -2095  -3257    392   -378    993   1394   1190    674   -768    338    -98    76
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -15 -10604 -11646   -894  -1115   -701  -1378  -7319  -6672 
    56   -878  -4177   1291     47  -4495  -3687    116  -4243   -878   1318    314    174  -3780    101   -388   -279   1620   -188  -4362  -3681    77
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -16 -10604 -11646   -894  -1115   -701  -1378  -7319  -6658 
    57    332    189   1866   1165  -4505    773  -2343  -4255    197  -1252   -463    423    626    549   -809  -1409  -1018  -3806  -4367  -1198    78
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -16 -10604 -11646   -894  -1115   -701  -1378  -7319  -6644 
    58   -892  -4184   1591    318  -4505    149  -2343  -4256   -126  -4200   -968   2412  -1002    438    227   -172    944  -3806  -4368  -3685    79
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -   -246 -10604  -2777   -894  -1115   -701  -1378  -7319  -6629 
    59  -1329    174   -757    767   -525   2620  -2154  -3997    337  -1733    798   -682  -3586  -1699     10   -197  -1000  -3565  -4143  -1070    80
     -   -147   -503    230     42   -379    400    102   -621    212   -463   -713    275    398     42     93    358    117   -372   -298   -252 
     -  -1238  -2595  -1320  -1857   -466  -1204   -821  -7319  -6615 
    60  -2057  -3525  -1901   -329  -3850   2119   1363  -3597   2009  -3538  -2614     31  -3118    745   -334   -879    654  -3149  -3701  -3025    88
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -17  -9783 -10825   -894  -1115   -268  -2562  -7319  -6600 
    61  -2879  -2722  -5113  -4484   1276  -4414    111   -726   -386    437    -50  -1066  -4465   -709  -1083   -567  -2819    801    569   3513    89
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -16 -10506 -11548   -894  -1115   -203  -2932  -7319  -6585 
    62   -748    210    -33  -4661   2244  -4517  -3387   -154  -1257  -1249    -79  -4156  -4567  -3888  -4068   -270   -499   -393  -3259   3581    90
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -16 -10605 -11647   -894  -1115   -701  -1378  -7319  -6570 
    63  -2973    218  -5314  -4678  -1099   -467  -3391   2396  -1381   1000    153   -899  -4570  -3897   1386  -3604  -2913   1112  -3257    275    91
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -17 -10605 -11647   -894  -1115   -701  -1378  -7319  -6555 
    64    712  -2898   -660  -4025   1011    361  -3235    260  -1186   -326   1009   -963   -432  -3491  -3780    856   1583   -803  -3342    273    92
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -17 -10605 -11647   -894  -1115   -701  -1378  -7319  -6539 
    65   -549  -4184    645   1588  -4504   1660  -2343  -4255   -796  -1102  -3273    -31      7   -633   -121   -737    680  -1389   1630  -3684    93
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -83 -10605  -4486   -894  -1115   -701  -1378  -7319  -6524 
    66   -298  -4130    161   1056  -4451   1605  -2289  -4202    438  -4146  -3219   1419  -3723   -486   1808   -200  -2596  -1076  -4313  -3630    94
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -17 -10539 -11581   -894  -1115   -263  -2587  -7319  -6508 
    67   -303  -4160    360    616  -4470  -3691   1917   -168    227   -456   -107  -2331    414    761    868   -549     38    706  -4350   -821    95
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -44 -10605  -5784   -894  -1115   -701  -1378  -7319  -6492 
    68   -324      4  -2598   -866  -1217   -822  -2358  -1795   1616  -1175  -3151   -812   -534     59   1409    -76   1405   -857  -4263   1318    96
     -   -147   -500    232     42   -381    398    105   -627    210   -464   -721    280    393     45     95    358    119   -370   -295   -250 
     -   -168  -3344 -11621   -393  -2067   -419  -1990  -7319  -6476 
    69  -1440    109  -6019  -5504   4127  -5317    790  -1078  -5097  -1202  -2848   -900  -5345  -4612  -4854  -4418  -3802  -3095  -3088   1330    99
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -18 -10606 -11648   -894  -1115   -701  -1378  -7319  -6459 
    70     70   1646   1226    358  -4499    205  -2344  -1253  -1926  -1839   -967   1545    818    358    611    882  -2650  -3801  -4364  -3683   100
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -18 -10606 -11648   -894  -1115   -701  -1378  -7319  -6443 
    71  -1218   -638    414  -2097  -4295   -423     60  -1827  -2008  -1801  -3145    661  -3822  -1962  -1116   2452   1944  -3625  -4262  -3621   101
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -18 -10606 -11648   -894  -1115   -701  -1378  -7319  -6426 
    72  -5783  -5235  -8195  -7617   -622  -7885  -6643   2054  -7396   2608    728  -7587   -377  -6310  -6953  -7193  -5650    329  -5352  -5508   102
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -18 -10606 -11648   -894  -1115   -701  -1378  -7319  -6409 
    73    565  -4171   -705   -967  -4486   -776    593  -4232   -547  -1032    481    806   1543   1630  -2437    396   -601   -211   1435    480   103
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -18 -10607 -11649   -894  -1115   -701  -1378  -7319  -6392 
    74   -124  -4175   1926   2547  -4488  -3694     34  -4234  -1939  -1173   -965  -2332  -1002   1851  -2445  -2602   -968  -3793  -4363   -558   104
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -19 -10607 -11649   -894  -1115   -701  -1378  -7319  -6375 
    75  -6325  -5649  -8690  -8081  -1212  -8579  -7091    722  -7904   3142   1192  -8327  -7377  -6484  -7287  -8024  -6133  -3977  -5433  -5722   105
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -19 -10607 -11649   -894  -1115   -701  -1378  -7319  -6358 
    76  -4820   1167  -7506  -7182  -5010  -7339  -7316   2750  -1473   -382  -3707  -6991  -7082  -7096  -7302  -6706  -4811   2854  -6723  -6200   106
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -19 -10607 -11649   -894  -1115   -701  -1378  -7319  -6340 
    77   -637  -4183    785   1664  -4504  -1528   1202  -4255   -758  -1224    539    723  -3778   1529   -403    388    -89  -1208    373    418   107
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -19 -10607 -11650   -894  -1115   -701  -1378  -7319  -6322 
    78  -1788  -3481  -1154  -1203   1121  -5023   4200  -3007  -4546   -831  -2689  -4410  -5074  -4179  -4480  -4114  -3550  -1166  -3181   3272   108
     -   -149   -500    233     43   -381    399    106   -626    210   -466   -720    275    394     45     96    359    117   -369   -294   -249 
     -    -20 -10608 -11650   -894  -1115   -701  -1378  -7319  -6304 
    79  -7240  -6153  -7608  -7964    271  -7493   3287  -6121  -7510  -1085  -5521  -6100  -7346  -1027  -6876  -6742  -7088  -6279  -2920   4367   109
     -      *      *      *      *      *      *      *      *      *      *      *      *      *      *      *      *      *      *      *      * 
     -      *      *      *      *      *      *      *  -7319      0 
//
