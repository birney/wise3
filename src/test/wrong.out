Score 1197
Position i:[-1] j:[287] State:[6] Score: 0
Position i:[0] j:[290] State:[0] Score: 1730
Position i:[1] j:[293] State:[0] Score: 1133
Position i:[2] j:[296] State:[0] Score: 612
Position i:[3] j:[299] State:[0] Score: 237
Position i:[0] j:[299] State:[7] Score: -2515
//
